apps/sdk-vite-integration/src/main.tsx

```
@@ -1,7 +1,7 @@
 import { StrictMode } from 'react'
 import { createRoot } from 'react-dom/client'
 import './index.css'
-import App from './App'
+import App from './App.js'
```

ESLint

[error] Insert `;`

(prettier/prettier)